railsbuild

```diff
@@ -24,12 +22,25 @@ if [ "$1" = "--help" ]; then
   exit 0
 fi
 
+# Option for not building
+[[ "$1" == "-n" ]] && {
+  NO_BUILD=y
+  shift
+} || NO_BUILD=
+
 if [ "$3" = "" ] || [ "$2" = "" ] || [ "$1" = "" ]; then
   echo "Error: railsbuild takes 3 parameters"
   print_help
   exit 1
 fi
 
+FEDORA_VERSION=$1
+OLD_RAILS_VERSION=$2
+NEW_RAILS_VERSION=$3
+
+# Set to default locale for the changelog date format
```

Comment:
This needs to be part of the offending script itself... so it works when the script is called as standalone program.